modules/ROOT/nav.adoc

```diff
@@ -77,16 +77,15 @@ include::{partialsdir}/attributes.adoc[]
 
 
 
-** xref:security.adoc[Security]
-*** xref:app-security.adoc[{app-security-service}]
-**** xref:app-security.adoc#introduction[Introduction]
-**** xref:app-security.adoc#setup[Setting Up]
-**** xref:app-security.adoc#monitoring[Monitoring App Security]
-*** xref:device-security.adoc[{device-security-service}]
-**** xref:device-security.adoc#introduction[Introduction]
-**** xref:device-security.adoc#setup[Setting Up]
-**** xref:device-security.adoc#device-trust[Performing Device Trust Checks]
-**** xref:device-security.adoc#cert-pinning[Certificate Pinning]
+** xref:security.adoc[Secure your app]
```

Comment:
Missed this on my first pass - does it make sense to change this as all other items in this list are actual mobile services, and this change stands out, ie:

![Screenshot from 2019-07-19 12-29-02](https://user-images.githubusercontent.com/12085451/61532292-459bec80-aa21-11e9-821a-bf182da743dc.png)

Reply:
good point, but will address later